src/main/java/org/jabref/gui/slr/StudyDatabaseItem.java

```diff
@@ -46,19 +61,12 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
-
         StudyDatabaseItem that = (StudyDatabaseItem) o;
-
-        if (isEnabled() != that.isEnabled()) {
-            return false;
-        }
-        return getName() != null ? getName().equals(that.getName()) : that.getName() == null;
+        return getName().equals(that.getName()) && (isEnabled() == (that.isEnabled()));
```

Comment:
```suggestion
        return Object.equals(getName(),that.getName()) && Object.equals(isEnabled(), that.isEnabled());
```
I prefer Object equals but this fine for me as well